cli/azd/test/samples/funcapp/infra/resources.bicep

```diff
@@ -1,11 +1,12 @@
-param name string
 param location string
+param resourceToken string
+param tags object
 
-resource function 'Microsoft.Web/sites@2021-03-01' = {
-  name: '${name}func'
+resource function 'Microsoft.Web/sites@2022-03-01' = {
+  name: 'func-${resourceToken}'
   location: location
   kind: 'functionapp,linux'
-
+  tags: union(tags, { 'azd-service-name': 'func' })
```

Comment:
Is `func` going to work here? This usually needs to be the actual azure resource name and `func` sounds too vague to be unique in azure.

Reply:
Func maps to the key in azure.yaml

Reply:
Yup - you're right